spec/features/edit_form_can_take_info_and_update_station_spec.rb

```diff
@@ -0,0 +1,35 @@
+require_relative '../spec_helper'
+
+describe "When user clicks the edit button" do
+  it "They are taken to a new page" do
+    Station.create(name: "Station Name",
+                  city_id: City.find_or_create_by(name: "Denver").id,
+                  dock_count_id: DockCount.find_or_create_by(dock_number: 15).id,
+                  installation_date_id: InstallationDate.find_or_create_by(installation_date: "8/7/2016").id)
+
+    visit('/stations')
+    find_link("Edit").click
+
+    expect page.has_current_path?('/stations/1/edit')
+  end
+
+  describe "They are able to fill out a form" do
+    it "and update a station" do
+      Station.create(name: "Station Name",
+                    city_id: City.find_or_create_by(name: "Denver").id,
+                    dock_count_id: DockCount.find_or_create_by(dock_number: 15).id,
+                    installation_date_id: InstallationDate.find_or_create_by(installation_date: '20090712').id)
```

Comment:
Sal has mentioned some things to look out for above, like the find_or_create_by and the DockCount table refactor, but in general I'm really impressed with the use of active record.